docs/upgrading/upgrading_to_v1.md

```diff
@@ -212,10 +212,20 @@ service_locator.set_storage_client(MemoryStorageClient())
 service_locator.set_storage_client(MemoryStorageClient())  # Raises an error
 ```
 
+## BasicCrawler changes
+
+### Renamed methods for opening storages
+`BasicCrawler.get_dataset` -> `BasicCrawler.open_dataset`
+`BasicCrawler.get_key_value_store` -> `BasicCrawler.open_key_value_store`
+
+### Added method for opening RequestQueue that uses configuration and storage client of the crawler
+`BasicCrawler.open_request_queue`
```

Comment:
Use words. This will be a page in the docs. It is not a changelog. Check out the other sections.